synapse/rest/consent/consent_resource.py

```diff
@@ -106,18 +107,14 @@ def __init__(self, hs):
 
         self._hmac_secret = hs.config.form_secret.encode("utf-8")
 
-    async def _async_render_GET(self, request):
-        """
-        Args:
-            request (twisted.web.http.Request):
-        """
+    async def _async_render_GET(self, request: Request) -> None:
         version = parse_string(request, "v", default=self._default_consent_version)
         username = parse_string(request, "u", default="")
         userhmac = None
         has_consented = False
         public_version = username == ""
         if not public_version:
-            args: Dict[bytes, List[bytes]] = request.args
+            args: Dict[bytes, List[bytes]] = request.args  # type: ignore
```

Comment:
Why do we need to ignore the type here ooi?

Reply:
We do this in many places -- Twisted has the wrong type for `request.args`.

Reply:
Ah, actually it isn't wrong, it's just missing. Without this we get:

```
synapse/rest/consent/consent_resource.py:118: error: Argument 1 to "parse_bytes_from_args" has incompatible type "Optional[Any]"; expected "Mapping[bytes, Sequence[bytes]]"  [arg-type]
```

Reply:
I have a feeling this might be the 154312th time I've asked that Q....

Reply:
> I have a feeling this might be the 154312th time I've asked that Q....

That's OK! I think it might be fixed in the next Twisted. I should double check / fix it.